lawnchair/res/values/strings.xml

```diff
@@ -556,6 +556,26 @@
     <string name="minus_one_unavailable">No feed apps installed</string>
     <string name="minus_one">Feed</string>
     <string name="feed_provider">Feed provider</string>
+    
+    <!-- AIDL Bridge -->
+    <string name="aidl_bridge_required">AIDL Bridge Required</string>
+    <string name="aidl_bridge_description">For feed compatibility on production builds</string>
+    <string name="aidl_bridge_install">Install</string>
+    <string name="aidl_bridge_info">Why AIDL Bridge?</string>
+    <string name="aidl_bridge_info_title">About AIDL Bridge</string>
+    <string name="aidl_bridge_info_message">This is due to the Google Feed API being more permissive on debug builds; as soon as you apply it to a production app, it will no longer work (hence the beta version not working). A bug is opened on Github for them to look into this, meanwhile the AIDLBridge fix is the best to use.</string>
```

Comment:
This is not accurate, Nightly builds functions the same as the GitHub variant under a different variant name.